src/components-examples/material/menu/menu-harness/menu-harness-example.spec.ts

```diff
@@ -0,0 +1,75 @@
+import {TestBed, ComponentFixture, waitForAsync, inject} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatMenuHarness} from '@angular/material/menu/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatMenuModule} from '@angular/material/menu';
+import {MenuHarnessExample} from './menu-harness-example';
+import {OverlayContainer} from '@angular/cdk/overlay';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+
+describe('MenuHarnessExample', () => {
+  let fixture: ComponentFixture<MenuHarnessExample>;
+  let loader: HarnessLoader;
+  let overlayContainer: OverlayContainer;
+
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatMenuModule, NoopAnimationsModule],
+        declarations: [MenuHarnessExample]
+      }).compileComponents();
+    }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(MenuHarnessExample);
+    fixture.detectChanges();
+    loader = TestbedHarnessEnvironment.loader(fixture);
+    inject([OverlayContainer], (oc: OverlayContainer) => {
+      overlayContainer = oc;
+    })();
+  });
+
+  afterEach(() => {
+    // Angular won't call this for us so we need to do it ourselves to avoid leaks.
+    overlayContainer.ngOnDestroy();
```

Comment:
I think that we should remove this one. It's a workaround for our own unit tests, but it might not be necessary for other people.